README.rst

```diff
@@ -108,154 +108,111 @@ Latest News
+For advanced users who want to train NeMo models from scratch or fine-tune existing NeMo models, we have a full suite of `example scripts <https://github.com/NVIDIA/NeMo/tree/main/examples>`_ that support multi-GPU/multi-node training.
 
-Key Features
-------------
+Documentation
+-------------
 
-* `Large Language Models <nemo/collections/nlp/README.md>`_
-* `Multimodal <nemo/collections/multimodal/README.md>`_
-* `Automatic Speech Recognition <nemo/collections/asr/README.md>`_
-* `Text to Speech <nemo/collections/tts/README.md>`_
-* `Computer Vision <nemo/collections/vision/README.md>`_
+* `NeMo Developer Docs < https://docs.nvidia.com/nemo-framework/user-guide/latest/nemotoolkit/index.html>`_
+* `NeMo Framework User Guide < https://docs.nvidia.com/nemo-framework/user-guide/latest/overview.html>`_
 
 Requirements
 ------------
 
-1) Python 3.10 or above
-2) Pytorch 1.13.1 or above
-3) NVIDIA GPU, if you intend to do model training
-
-Developer Documentation
------------------------
+* Python 3.10 or above
+* Pytorch 1.13.1 or above
+* NVIDIA GPU (if you intend to do model training)
 
-.. |main| image:: https://readthedocs.com/projects/nvidia-nemo/badge/?version=main
-  :alt: Documentation Status
-  :scale: 100%
-  :target: https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/
-
-.. |stable| image:: https://readthedocs.com/projects/nvidia-nemo/badge/?version=stable
-  :alt: Documentation Status
-  :scale: 100%
-  :target:  https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/
-
-+---------+-------------+------------------------------------------------------------------------------------------------------------------------------------------+
-| Version | Status      | Description                                                                                                                              |
-+=========+=============+==========================================================================================================================================+
-| Latest  | |main|      | `Documentation of the latest (i.e. main) branch. <https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/>`_                  |
-+---------+-------------+------------------------------------------------------------------------------------------------------------------------------------------+
-| Stable  | |stable|    | `Documentation of the stable (i.e. most recent release) branch. <https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/>`_ |
-+---------+-------------+------------------------------------------------------------------------------------------------------------------------------------------+
-
-
-Getting help with NeMo
+Install NeMo Framework
 ----------------------
-FAQ can be found on NeMo's `Discussions board <https://github.com/NVIDIA/NeMo/discussions>`_. You are welcome to ask questions or start discussions there.
```

Comment:
Add this back

Reply:
The information was added to the end of the file

Reply:
OK move to bottom of file. Change to Discussions Board.